src/service_name.json

```diff
@@ -0,0 +1,422 @@
+[
+  {
+    "Command": "az ai-did-you-mean-this",
+    "Description": "Recommend recovery options on failure.",
+    "AzureServiceName": "Microsoft AI",
+    "ConversationalServiceName": "Microsoft AI",
+    "URL": ""
+  },
+  {
+    "Command": "az ai-examples",
```

Comment:
how to maintain this file? what's process when new extension is released?

Reply:
Manual. We can add a linter rule to force it.
I added a new column "ExtensionName". It is required when "AzureCLIReferenceType" is Extension or "hasExtension" is 1. This column is used to check if all extensions have records.